test/proxy/beacon/BeaconProxy.test.js

```diff
@@ -34,7 +34,7 @@ describe('BeaconProxy', function () {
     it('non-compliant beacon', async function () {
       const badBeacon = await ethers.deployContract('BadBeaconNoImpl');
 
-      await expect(this.newBeaconProxy(badBeacon, '0x')).to.be.revertedWithoutReason;
+      await expect(this.newBeaconProxy(badBeacon, '0x')).to.be.reverted;
```

Comment:
Here the revert is:
- BadBeaconNoImpl does not implement `implementation()`
- ERC1967 utils try to get that in `_setBeacon`
- the call fail (no impelmentation of the selector + no fallback)

Reply:
It was previously:

```
await expect(this.newBeaconProxy(badBeacon, '0x')).to.be.revertedWithoutReason
```

Which seemed correct to me but I'm realizing it was missing the parenthesis:

```
await expect(this.newBeaconProxy(badBeacon, '0x')).to.be.revertedWithoutReason();
```

We can keep it with the explicit `revertedWithoutReason()` and keep the comments.